OpenRA.Game/Graphics/SpriteRenderer.cs

```diff
@@ -50,7 +50,7 @@ public void Flush()
 
 				renderer.Context.SetBlendMode(currentBlend);
 				shader.PrepareRender();
-				renderer.DrawBatch(vertices, nv, PrimitiveType.TriangleList);
+				renderer.DrawBatch(ref vertices, nv, PrimitiveType.TriangleList);
```

Comment:
Can we please add a comment `// PERF: The renderer may choose to replace vertices with a different temporary buffer.`?